packages/react/src/components/LoaderOverlay/index.tsx

```diff
@@ -7,37 +7,65 @@ import AxeLoader from './axe-loader';
 interface LoaderOverlayProps extends React.HTMLAttributes<HTMLDivElement> {
   variant?: 'large' | 'small';
   label?: string;
+  focus?: boolean;
+  children?: React.ReactNode;
 }
 
 const LoaderOverlay = React.forwardRef<HTMLDivElement, LoaderOverlayProps>(
-  ({ className, variant, label, ...other }: LoaderOverlayProps, ref) => (
-    <div
-      className={classNames(
-        'Loader__overlay',
-        className,
-        variant === 'large'
-          ? 'Loader__overlay--large'
-          : variant === 'small'
-          ? 'Loader__overlay--small'
-          : ''
-      )}
-      ref={ref}
-      {...other}
-    >
-      <div className="Loader__overlay__loader">
-        <Loader variant={variant} />
-        <AxeLoader />
+  (
+    {
+      className,
+      variant,
+      label,
+      children,
+      focus,
```

Comment:
This prop is a bit ambiguous as it doesn't describe what the prop does. I would like to think we could give this a name that's a little more self documenting, like `focusOnInitialRender`.